atoma-p2p/src/service.rs

```diff
@@ -224,9 +230,27 @@ impl AtomaP2pNode {
         }
         let mut metrics_registry = libp2p::metrics::Registry::default();
 
-        let local_key = identity::Keypair::generate_ed25519();
+        let local_key = std::thread::spawn(move || {
```

Comment:
Why spawning a new thread for this ? I believe reading the local key from the file should be under ms ?

Reply:
I wanted to avoid making the start function `async` and so it's not possible to block in the current runtime, as that would be creating a runtime within the runtime, thus spawning a new thread is a better alternative in my view